<!--
Is this the designed type material for a new taxon?                     typn                        BOOLEAN
Strain or MAG or SAG from which the reference taxon was obtained        otyp                        VARCHAR(30)
Designated type strain name/MAG/SAG                                     type                        VARCHAR(30)
16S rRNA gene accession number                                          sixteensr                   VARCHAR(20)  
Sequencing depth                                                        seqdepth                    FLOAT
23S rRNA gene accession number                                          twentythreesr               VARCHAR(20)
MAG accession number(RefSeq/ENA)                                        gare(MV)                    VARCHAR(20)
MAG accession number(other)                                             binn                        VARCHAR(20)*[no GEN]
Genome status                                                           gsta(MV)                    ENUM("complete","partial","draft")
Completness                                                             completeness                FLOAT
Contaminations                                                          contamination               FLOAT
Method used to estimate the integrity and contamination of the sample   method(MV)                  INT
Estimated or actual genome/metagenome size. Unit bp (base pairs)        gsiz(METAGENOME),                                   INT
Molar percentage of Guanine and Cytosine in DNA.                        ggcm(MV)                    FLOAT
Binning software used                                                   bins                        INT*[no GEN]
Params used with bins                                                   binsparams                  VARCHAR(80)*[no GEN]
Biotic relationship with other organisms.                               bior                        VARCHAR(100)
If it is a symbiont, with which host does it interact?                  host                        VARCHAR(100)
Known pathogenicity.                                                    path                        VARCHAR(100)
-->
<div class="row">
  <!-- Is this the designed type material for a new taxon? -->
  <div class="col">
    <label for="typn" class="col-form-label"
      >Is this the designed type material for a new taxon?</label
    >
    <input type="checkbox" id="typn" />
  </div>
</div>
<div class="row">
  <!-- Strain or MAG or SAG from which the reference taxon was obtained -->
  <div class="col">
    <label for="otyp" class="col-form-label"
      >Reference Strain/MAG/SAG<br />
      taxon
    </label>
    <input type="text" maxlength="30" class="form-control" id="otyp" />
  </div>
  <!-- Designated type strain name/MAG/SAG -->
  <div class="col">
    <label for="type" class="col-form-label"
      >Designated type strain name/MAG/SAG</label
    >

    <input type="text" maxlength="30" class="form-control" id="type" />
  </div>
  <!-- 16S rRNA gene accession number -->
  <div class="col">
    <label for="sixteensr" class="col-form-label"
      >16S rRNA gene <br />accession number</label
    >
    <input type="text" maxlength="20" class="form-control" id="sixteensr" />
  </div>
  <!-- 23S rRNA gene accession number -->
  <div class="col">
    <label for="twentythreesr" class="col-form-label"
      >23S rRNA gene <br />accession number</label
    >

    <input type="text" maxlength="20" class="form-control" id="twentythreesr" />
  </div>
</div>
<div class="row">
  <!-- Sequencing depth -->
  <div class="col">
    <label for="seqdepth" class="col-form-label">Sequencing depth</label>

    <input type="number" step="any" class="form-control" id="seqdepth" />
  </div>
  <!-- MAG accession number (RefSeq/ENA) -->
  <div class="col">
    <label for="gare" class="col-form-label"
      >MAG accession number (RefSeq/ENA)</label
    >

    <input type="text" maxlength="20" class="form-control" id="gare" />
  </div>

  <!-- MAG accession number (other) -->
  <div class="col">
    <label for="binn" class="col-form-label"
      >MAG accession number (other)</label
    >

    <input type="text" maxlength="20" class="form-control" id="binn" />
  </div>
</div>
<div class="row">
  <!-- Genome status -->
  <div class="col">
    <label for="gsta" class="col-form-label">Genome status</label>

    <select class="form-select" id="gsta">
      <option value="complete">Complete</option>
      <option value="partial">Partial</option>
      <option value="draft">Draft</option>
    </select>
  </div>

  <!-- Completeness -->
  <div class="col">
    <label for="completeness" class="col-form-label">Completeness</label>
    <input type="number" step="any" class="form-control" id="completeness" />
  </div>

  <!-- Contaminations -->
  <div class="col">
    <label for="contamination" class="col-form-label">Contaminations</label>
    <input type="number" step="any" class="form-control" id="contamination" />
  </div>
  <!-- Method used to estimate the integrity and contamination of the sample -->
  <div class="col-6">
    <label for="method" class="col-form-label"
      >Method used to estimate integrity and contamination</label
    >
    <input type="number" class="form-control" id="method" />
  </div>
</div>
<div class="row">
  <!-- Estimated or actual genome/metagenome size (bp) -->
  <div class="col">
    <label for="gsiz" class="col-form-label"
      >Estimated or actual genome/metagenomesize</label
    >
    <div class="input-group">
      <input type="number" class="form-control" id="gsiz" />
      <div class="input-group-append">
        <span class="input-group-text">bp</span>
      </div>
    </div>
  </div>

  <!-- Molar percentage of Guanine and Cytosine in DNA -->
  <div class="col">
    <label for="ggcm" class="col-form-label"
      >Molar % of Guanine and Cytosine in DNA</label
    >
    <div class="input-group">
      <input type="number" step="any" class="form-control" id="ggcm" />
      <div class="input-group-append">
        <span class="input-group-text">%</span>
      </div>
    </div>
  </div>
  <!-- Binning software used -->
  <div class="col">
    <label for="bins" class="col-form-label">Binning software <br />used</label>

    <input type="number" class="form-control" id="bins" />
  </div>

  <!-- Parameters used with bins -->
  <div class="col">
    <label for="binsparams" class="col-form-label"
      >Parameters used with <br />bins</label
    >

    <input type="text" maxlength="80" class="form-control" id="binsparams" />
  </div>
</div>
<div class="row">
  <!-- Biotic relationship with other organisms -->
  <div class="col">
    <label for="bior" class="col-form-label"
      >Biotic relationship with other organisms</label
    >
    <input type="text" maxlength="100" class="form-control" id="bior" />
  </div>

  <!-- If it is a symbiont, with which host does it interact? -->
  <div class="col">
    <label for="host" class="col-form-label"
      >Interction Host(Only symbiont)</label
    >
    <input type="text" maxlength="100" class="form-control" id="host" />
  </div>

  <!-- Known pathogenicity -->
  <div class="col">
    <label for="path" class="col-form-label">Known pathogenicity</label>

    <input type="text" maxlength="100" class="form-control" id="path" />
  </div>
</div>
